sql/moz-fx-data-shared-prod/monitoring/stable_table_sizes/view.sql

```diff
@@ -4,4 +4,4 @@ AS
 SELECT
   *
 FROM
-  `moz-fx-data-shared-prod.monitoring_derived.stable_table_sizes_v1`
+  `moz-fx-data-shared-prod.monitoring_derived.stable_and_derived_table_sizes_v1`
```

Comment:
I would do this here
```suggestion
  `moz-fx-data-shared-prod.monitoring_derived.stable_and_derived_table_sizes_v1`
  WHERE destination_dataset LIKE '%_stable'
```

And a new view in a separate file in `monitoring/stable_and_derived_table_sizes/view.sql`

```
SELECT
  *
FROM
  `moz-fx-data-shared-prod.monitoring_derived.stable_and_derived_table_sizes_v1`
```